examples/window-app/common/src/WindowApp.cpp

```diff
@@ -113,7 +117,11 @@ WindowApp::Cover * WindowApp::GetCover(chip::EndpointId endpoint)
 CHIP_ERROR WindowApp::Init()
 {
     // Initialize device attestation config
+#ifdef DEVICE_ATTESTATION_CREDENTIALS
```

Comment:
This is a platform specific override. This does not scale and the name of the define doesn't convey that this is EFR32 only